From b900753b66d99bf0c5638bbba919eb56b83d884c Mon Sep 17 00:00:00 2001
From: Frank Wang <frank.wang@rock-chips.com>
Date: Mon, 19 Sep 2016 17:22:58 +0800
Subject: [PATCH] arm64: dts: rockchip: add usb2 phy-phandle for ohci on rk3366

This adds support usb2.0 phy-phandle for ohci controller on rk3366.

Change-Id: I9b5e27636e7574669ba01e4302c741d8895c68ff
Signed-off-by: Frank Wang <frank.wang@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3366.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3366.dtsi b/arch/arm64/boot/dts/rockchip/rk3366.dtsi
index 749cdf12e67b..f8d8c8111d3e 100644
--- a/arch/arm64/boot/dts/rockchip/rk3366.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3366.dtsi
@@ -492,6 +492,8 @@
 		interrupts = <GIC_SPI 25 IRQ_TYPE_LEVEL_HIGH>;
 		clocks = <&cru SCLK_USBPHY480M>, <&cru HCLK_HOST>;
 		clock-names = "usbphy_480m", "hclk_host0";
+		phys = <&u2phy_host>;
+		phy-names = "usb";
 		status = "disabled";
 	};
 
-- 
2.35.3

